src/main/resources/swagger/api-docs.yaml

```diff
@@ -154,11 +134,6 @@ paths:
       operationId: resourceRoles
       tags:
         - Resources
-      security:
```

Comment:
are these implied now?

Reply:
yes, they always have been but we have just been copy pasting